src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs

```diff
@@ -102,6 +112,14 @@ public bool ImportRequestContext()
             return false;
         }
 
+        public void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
So the sequence token creation must be removed from the [AzureQueueBatchContainer.cs#L91](https://github.com/dotnet/orleans/blob/fc0af9079413505de6c5dad928a266fccda74d68/src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs#L91) and instead it should be serialized among other fields.